conduit/pipeline/pipeline.go

```diff
@@ -431,8 +467,250 @@ func addMetrics(block data.BlockData, importTime time.Duration) {
 	metrics.ImportedTxnsPerBlock.Observe(float64(len(block.Payset)) + float64(innerTxn))
 }
 
+func (p *pipelineImpl) importerHandler(importer importers.Importer, roundChan <-chan uint64, blkOutChan pluginChannel) {
+	p.wg.Add(1)
+	go func() {
```

Comment:
Why did you decide to launch the goroutines inside of each of the plugin functions instead of writing them as regular functions and having `Start` launch all of them as goroutines?

Reply:
I can see though, that there could be a convention by which all go-routines launched from a function ought to be launched inside that man function with `go xyz()`... If we want to follow such a convention, I can refactor.